lib/util.js

```diff
@@ -348,9 +348,30 @@ exports.addFormatMeta = format => {
  * @param {string} html
  * @returns {string}
  */
-exports.stripHTML = html => html
+exports.stripHTML = html => decodeURIComponent(html)
```

Comment:
I'm getting a `URIError: URI malformed` error from this

try calling it after this function is called